### PR TITLE
[tests-only][full-ci] test for overwriting a file with virus content in group share

### PR DESCRIPTION
**Pull request by ghost**
## Description
This PR adds the API test scenario for overwrite file with virus content in group share
* Scenario Outline: try to overwrite a file with the virus content in group share
* Scenario: try to overwrite a file with the virus content in group share spaces

## Related Issue
* Part of: https://github.com/owncloud/ocis/issues/6255

#### test environment:
* locally
* CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [X] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [X] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/apiAntivirus/antivirus.feature [hunk 1]
```diff
@@ -335,3 +335,41 @@ Feature: antivirus
       | old              |
       | new              |
       | spaces           |
+
+  @issue-enterprise-5709
```
Comment (SwikritiT) [inline tests/acceptance/features/apiAntivirus/antivirus.feature:339]: We don't need this issue tag here. Normally we put issue tags on the scenarios that were written to cover a bug that was fixed for a documentation purpose or a bug that still exists to indicate that this scenario might not pass locally. 
```suggestion

```

File: tests/acceptance/features/apiAntivirus/antivirus.feature [hunk 2]
```diff
@@ -335,3 +335,41 @@ Feature: antivirus
...
+      | message                                                                   |
+      | Virus found in test.txt. Upload not possible. Virus: Win.Test.EICAR_HDB-1 |
+    And the content of file "/test.txt" for user "Alice" should be "hello"
+    And the content of file "Shares/test.txt" for user "Brian" should be "hello"
+    Examples:
+      | dav-path-version |
+      | old              |
+      | new              |
+
+  @issue-enterprise-5709
```
Comment (SwikritiT) [inline tests/acceptance/features/apiAntivirus/antivirus.feature:360]: same here 
```suggestion
```

File: tests/acceptance/features/apiAntivirus/antivirus.feature [hunk 3]
```diff
@@ -335,3 +335,41 @@ Feature: antivirus
       | old              |
       | new              |
       | spaces           |
+
+  @issue-enterprise-5709
+  Scenario Outline: try to overwrite a file with the virus content in group share
+    Given using <dav-path-version> DAV path
+    And user "Brian" has been created with default attributes and without skeleton files
+    And group "group1" has been created
+    And user "Brian" has been added to group "group1"
```
Comment (SwikritiT) [inline tests/acceptance/features/apiAntivirus/antivirus.feature:344]:
```suggestion
    And user "Alice" has been added to group "group1"
```

Comment (SwikritiT) [inline tests/acceptance/features/apiAntivirus/antivirus.feature:344]:
```suggestion
    And user "Brian" has been added to group "group1"
    And user "Alice" has been added to group "group1"
```

File: tests/acceptance/features/apiAntivirus/antivirus.feature [hunk 4]
```diff
@@ -335,3 +335,41 @@ Feature: antivirus
...
+      | dav-path-version |
+      | old              |
+      | new              |
+
+  @issue-enterprise-5709
+  Scenario: try to overwrite a file with the virus content in group share spaces
+    Given using spaces DAV path
+    And user "Brian" has been created with default attributes and without skeleton files
+    And group "group1" has been created
+    And user "Brian" has been added to group "group1"
```
Comment (SwikritiT) [inline tests/acceptance/features/apiAntivirus/antivirus.feature:365]:
```suggestion
    And user "Brian" has been added to group "group1"
     And user "Alice" has been added to group "group1"
```

File: tests/acceptance/features/apiAntivirus/antivirus.feature [hunk 5]
```diff
@@ -335,3 +335,41 @@ Feature: antivirus
...
+      | Virus found in test.txt. Upload not possible. Virus: Win.Test.EICAR_HDB-1 |
+    And the content of file "/test.txt" for user "Alice" should be "hello"
+    And the content of file "Shares/test.txt" for user "Brian" should be "hello"
+    Examples:
+      | dav-path-version |
+      | old              |
+      | new              |
+
+  @issue-enterprise-5709
+  Scenario: try to overwrite a file with the virus content in group share spaces
```
Comment (SwikritiT) [inline tests/acceptance/features/apiAntivirus/antivirus.feature:361]:
```suggestion
  Scenario: try to overwrite a file with the virus content in group share using spaces dav endpoint
```

**Review by SwikritiT — APPROVED** [review]
LGTM

**Review by grgprarup — COMMENTED** [review]
lgtm :+1:

**Pull request merged**